etl/steps/data/garden/gcp/2022-09-29/global_carbon_budget_additional.meta.yml

```diff
@@ -0,0 +1,81 @@
+dataset:
+  title: Global Carbon Budget (Global Carbon Project, v2021b)
+  sources:
+    -
+      name: Global Carbon Project (2021)
+      published_by: Global Carbon Budget - Global Carbon Project (2021)
+      publisher_source: Global Carbon Project
+      description: |
+        The Global Carbon Budget dataset is available <a href="https://www.icos-cp.eu/science-and-impact/global-carbon-budget/2021">here</a> and <a href="https://doi.org/10.5281/zenodo.5569235">here</a>.
+
+        Variables include each country, region and World Bank income group's share of the global population; production-based (territorial); and consumption-based (trade-adjusted) carbon dioxide emissions. 
+
+        This was calculated by Our World in Data based on CO₂ figures produced by the Global Carbon Project. This is given as production (territorial) emissions in addition to trade-adjusted consumption-based emissions. Consumption-based emissions are national or regional emissions which have been adjusted for trade (i.e. territorial/production emissions minus emissions embedded in exports, plus emissions embedded in imports). If a country's consumption-based emissions are higher than its production emissions it is a net importer of carbon dioxide.
+
+        Note that consumption-based emissions are not available for all countries; although those without complete data are a small fraction (3%) of the global total. Each country's share of world emissions are based on the share of the global total minus categories termed 'bunkers' and 'statistical differences' (which include cross-boundary emissions such as international travel and shipping.
+
+        Calculation of each country's share of the global population is calculated using national and global population figures from the UN World Population Prospects (UNWPPP, 2018). 
+
```

Comment:
Small typo on UNWPP - extra P!

Reply:
And should it be UNWPP 2022?

Reply:
Good point, I've changed it, since that was not even true (I'm referring now to our population dataset, based on different sources).